lib/embulk/input/twitter_ads_analytics.rb

```diff
@@ -280,6 +317,20 @@ def entity_plural(entity)
           "FUNDING_INSTRUMENTS"
         end
       end
+
+      private
+
+      def exp_backoff_sec(response:, retries:)
```

Comment:
This method is no longer an exponential backoff.

Reply:
So, please rename the method name.

Reply:
fixed by https://github.com/trocco-io/embulk-input-twitter_ads_analytics/pull/6/commits/255ad1528a4205f3139fc0063be2bc875be6f544